.github/workflows/sentry.yml

```diff
@@ -0,0 +1,26 @@
+name: New Sentry Release
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout release
+      - uses: actions/checkout@v3
+        with: 
+          ref: release
+      - name: Sentry Release
+        uses: getsentry/action-release@v1.7.0
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+          SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
+        with:
+          environment: production
+          # uses the release PR title as a version
+          version: ${{ github.event.pull_request.title }}
+          # This can safely be removed after one succesful release - previous release hash won't be found in repo as we're squashing this
```

Comment:
Seems like we would always want this due to squash merge? Unless I am misunderstanding

Reply:
Actually valid point! This was originally written with a push directive as well which I've removed because of the PR title shenanigans

Reply:
5772c9865e87d2caf62bf5e7f707cffb15af9ad